todo_app/app.py

```diff
@@ -1,34 +1,35 @@
 from flask import Flask, render_template, request, redirect
-from todo_app.data.trello_config import TrelloConfig
+
+from todo_app.data.mongo_items import MongoItems
+from todo_app.data.mongo_config import MongoConfig
 
 from todo_app.view_model import ViewModel
-from todo_app.data.trello_items import TrelloItems
 
 def create_app():
     app = Flask(__name__)
-    trello_items = TrelloItems(TrelloConfig())
+    mongo_items = MongoItems(MongoConfig())
 
     @app.route('/')
     def index():
-        items = sorted(trello_items.get_items(), key=lambda item: item.status == 'Done')
+        items = sorted(mongo_items.get_items(), key=lambda item: item.status == 'Done')
```

Comment:
This change being so simple here is a good sign your interface for this mongo_items/trello_items file is well designed 👍